client/consul.go

```diff
@@ -124,6 +147,7 @@ func (c *ConsulService) Register(task *structs.Task, allocID string) error {
 	return mErr.ErrorOrNil()
 }
 
+// Stops tracking a task for changes to it's services and checks
```

Comment:
Can you fix the comments that don't match the Go style.

Also shouldn't these two mention that it tracks AND registers/deregisters